projects/developer/src/app/configuration/recipe-types/component.html

```diff
@@ -3,7 +3,7 @@
         <h2><i class="fa fa-cubes"></i> Recipe Types ({{ totalRecords }})</h2>
         <div class="margin-left-md">
             <button pButton type="button" icon="fa fa-plus" (click)="createNewRecipe()"
-                    pTooltip="Create Recipe Type"></button>
+                    pTooltip="Create Recipe Type" *ngIf="globals.is_staff"></button>

```

Comment:
So I think you'll also need to create a router guard. See Rachel's pending-changes.guard.ts for the structure. This should be added to the routes to prevent navigation to them. You can either just redirect the user back to the home page, or create a new "not allowed"/"forbidden" page and redirect them to that.

Reply:
Makes sense.  I will do that.